dask/dataframe/io/parquet/fastparquet.py

```diff
@@ -71,6 +77,9 @@ def _paths_to_cats(paths, scheme):
     return {k: list(v) for k, v in cats.items()}
 
 
+paths_to_cats = paths_to_cats or _paths_to_cats
```

Comment:
Is there any reason to fall back to the old implementation here for older fastparquet versions?

If so, I think the `try` import block is fine.

If not, maybe `_paths_to_cats` should just be modified to match the definition from `fastparquet.api`?  In an ideal world we would definitely want to avoid redundant code.  However, since we *cannot* do this yet, maybe it is sufficient to make a note/comment that we can import `paths_to_cats` from `fastparquet.api` once version `>0.3.2` is a requirement?  (cc @martindurant @TomAugspurger for thoughts)

Reply:
Good point, I should at least copy paste the new implementation

Reply:
I have in-lined the new function for now.